src/Symfony/Component/Messenger/Exception/NestedExceptionsTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Exception;
+
+/**
+ * @author Jeroen <https://github.com/Jeroeny>
+ */
+trait NestedExceptionsTrait
+{
+    /**
+     * @return \Throwable[]
+     */
+    public function getNestedExceptions(bool $recursive = true): array
```

Comment:
ok, so if we go with a new method, what about this signature? (and deprecate getNestedExceptionOfClass)
`getWrappedExceptions(string $class = null, bool $recursive = false)`

Reply:
That should be fine too, I've committed the changes. Lmk what you think.